Resources/Prototypes/Entities/Objects/Devices/Syndicate_Gadgets/reinforcement_teleporter.yml

```
@@ -57,7 +57,7 @@
   - type: GhostRole
     name: ghost-role-information-syndicate-monkey-reinforcement-name
     description: ghost-role-information-syndicate-monkey-reinforcement-description
-    rules: ghost-role-information-rules-default-familiar
+    rules: ghost-role-information-familiar-rules
```

```suggestion
    rules: ghost-role-information-syndicate-monkey-reinforcement-rules
```
Use them here.
Same with the others below, they are not familiars, they are team antagonists.
Use the correct strings from `ghost-role-component.ftl`

---

With the new commit, all Syndicate reinforcements now get the same rule locale. The previous 3 all said the exact same thing